src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -265,6 +265,10 @@ private static ListSelectionListener createDescriptionPanelUpdatingSelectionList
       final JList<String> gamesList, final List<DownloadFileDescription> maps, final MapAction action,
       final JLabel mapSizeLabelToUpdate) {
     return e -> {
+      if (e.getValueIsAdjusting()) {
+        return;
+      }
+
```

Comment:
I'd enclose the following code in the opposite if statement instead.
Looks better IMO